### PR TITLE
CW hacked

### PR DESCRIPTION
**Pull request by aepaysinger**

### CODE REVIEW

File: code_challenges/code_wars/hacked.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+def find_hack(student_grades):
+    checked_scores = {}
+    hacked_students = []
+    for student in student_grades:
+        checked_scores[student[0]] = [student[1]]
```
Comment (W-Ely) [inline code_challenges/code_wars/hacked.py:5]: How can you avoid student[0] and student[1] here?

**Review by W-Ely — COMMENTED** [review]
I can't review this without knowing what it is supposed to do.

File: code_challenges/code_wars/hacked.py (hunk 1 above)
Comment (aepaysinger) [reply code_challenges/code_wars/hacked.py]: pull it out into variables?

File: code_challenges/code_wars/hacked.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+def find_hack(student_grades):
+    checked_scores = {}
+    hacked_students = []
+    for student in student_grades:
+        name, old_score, grades = [category for category in student]
```
Comment (W-Ely) [inline code_challenges/code_wars/hacked.py:5]:
```suggestion
    for name, old_score, grades in student_grades:
```

**Review by W-Ely — APPROVED** [review]

**Pull request merged**